language/testing-infra/transactional-test-runner/src/tasks.rs

```diff
@@ -24,10 +24,14 @@ pub enum RawAddress {
 }
 
 fn parse_address_literal(s: &str) -> Result<AccountAddress> {
-    let (number, _number_format) = move_compiler::shared::parse_u128(s)
-        .map_err(|e| anyhow!("Failed to parse address. Got error: {}", e))?;
-
-    Ok(AccountAddress::new(number.to_be_bytes()))
+    match move_compiler::shared::parse_address(s) {
+        Some((array, _)) => Ok(AccountAddress::new(array)),
+        None => Err(anyhow!(
+            "Failed to parse address {} to AccountAddress with Length {}",
+            s,
+            AccountAddress::LENGTH
+        )),
```

Comment:
I think it might be a little more syntactically pleasing to use
```
AccountAddress::new(array).ok_or_else(|| Err(anyhow!()))....

not sure if ok_or passes lints.